assets/script.js

```diff
@@ -19,7 +19,29 @@ function startTimer(seconds) {
       timer--;
     }
   }, 1000);
+
+}
+
+function displayContent(type) {
+    var container = document.getElementById('document.container');
```

Comment:
This should be a global variable at the top of the page, other functions might need to access the container.

Also, check the syntax on document.getElementById:
currently you're not getting any elements.